Docker/test/selenium/specs/repo/special-version.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const assert = require( 'assert' );
+
+describe( 'Special:Version', function () {
+
+	const extensions = {
+		wikibase: [ 'WikibaseClient', 'WikibaseRepository' ],
+		other: [ 'OAuth' ]
+	};
+
+	// should be disabled for dynamic tests
+	// eslint-disable-next-line mocha/no-setup-in-describe
+	[ 'wikibase', 'other' ].forEach( ( extensionPackage ) => {
```

Comment:
At first I didn't see the connection between this and the forEach below. Maybe using `Object.keys( extensions )` here would make it clearer that we are iterating over two levels of the same thing here? Feel free to disregard this, if this doesn't sound sensible to you.

Reply:
sounds reasonable!